salt/engines/hipchat.py

```diff
@@ -28,7 +28,7 @@
                    list_jobs:
                        cmd: jobs.list_jobs
                    list_commands:
-                       cmd: pillar.get salt:engines:hipchat:valid_commands target=saltmaster
+                       cmd: pillar.get salt:engines:hipchat:valid_commands target=saltmaster expr_form=list
```

Comment:
`expr_form` will be deprecated as of Nitrogen and `tgt_type` should be used instead.
I notice this engine did not have any of them before, so I think we should introduce the recommended arg, which is `tgt_type`.